docs/docker-integration/cloudformation.rst

```diff
@@ -0,0 +1,167 @@
+            </div>
+		</div>
+		<div class="step-stages__step">
+			<img src="../_images/12-cloudformation-settings.png" alt="Optionally fill in volumehub settings"/>
+            <span>Optionally, <a href="https://clusterhq.com/volumehub/" target="_blank">register for a Volume Hub account</a>. This provides a web interface to see what's going on in your cluster.</span>
+           <span>Once you're logged in to the Volume Hub, <a href="https://volumehub.clusterhq.com/v1/token" target="_blank">get your Volume Hub token here</a> and copy the <code>&lt;YourToken&gt;</code> part from <code>{"token": "&lt;YourToken&gt;"}</code> into the <code>VolumeHubToken</code> field.</span>
+		</div>
+		<div class="step-stages__step">
+			<img src="../_images/13-cloudformation-create.png" alt="Click create"/>
+			<span>Click "Next" twice and then click "Create" to create your cluster.</span>
+		</div>
+	</div>
+
+	<div class="step-stages step-stages--3up">
+		<div class="step-stages__excerpt">
+			<h2 class="step-stages__heading">The waiting bit!</h2>
+			<p>Wait for the cluster to come up. This can take 5-10 minutes.</p>
+		</div>
+		<div class="step-stages__step first">
+			<img src="../_images/21-refresh.png" alt="Refresh button on CloudFormation console"/>
+			<span>The stack may not show up immediately. Click the refresh button a few times to see it show up.</span>
+		</div>
+		<div class="step-stages__step">
+			<img src="../_images/22-create-in-progress.png" alt="Stack create in progress message"/>
+			<span>Once the stack shows up, it will stay in CREATE_IN_PROGRESS state for 5-10 minutes. Wait for it to transition to...</span>
+		</div>
+		<div class="step-stages__step">
+			<img src="../_images/23-create-complete.png" alt="Stack create create complete message"/>
+			<span>... CREATE_COMPLETE state.</span>
+		</div>
+	</div>
+
+	<div class="step-stages step-stages--3up">
+		<div class="step-stages__excerpt">
+			<h2 class="step-stages__heading">Step 3</h2>
+			<p>Verifying your installation.</p>
+            <p>Click on the "Outputs" tab for your stack. If this is not visible, click the drop down icon on the current tab.</p>
+            <p>The values displayed on this tab will be used for verifying your installation and also any tutorials you go through.</p>
+			<img src="../_images/31-stack-outputs.png" alt="Stack outputs in CloudFormation" style="margin: 2em 0;"/>
+            <p>Now open a Terminal window, and run the following commands to log in and verify your cluster is working.</p>
+            <p>Where a command includes a string like <code>&lt;ClientNodeIP&gt;</code>, use the corresponding value from the Outputs tab.</p>
+            <p>Where a command has <code>&lt;KeyPath&gt;</code> this should be the path on your machine to the <code>.pem</code> file you downloaded in Step 1, for example: <code>~/Downloads/flocker-test.pem</code>.</p>
```

Comment:
:question: I suggest that these can be moved below the code block.
Most users will understand that they need to replace the `<VARIABLE>` bits.